tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -234,7 +239,7 @@ namespace tensorflow {
 // -------------------------------------------
 // Consider BiasAddGrad op as:
 //
-//           O = MklConv2D(A, A_m, B, B_m, C, C_m)
+//           O = MklConv2D(A, B, C, A_m, B_m, C_m)
 //           P = BiasAddGrad(O)
 //
 // Then we rewrite is as:
```

Comment:
I fixed error in line 221. As per my understanding of English, I think line 227 is correct. Feel free to let me know if I am wrong.